Makefile.inc

```diff
@@ -59,6 +59,9 @@ endif
 
 ifneq (,$(filter ovn,$(_using)))
 PRELOAD_IMAGES += submariner-networkplugin-syncer
+SETTINGS ?= $(DAPPER_SOURCE)/.shipyard.e2e.ovn.yml
+else
+SETTINGS ?= $(DAPPER_SOURCE)/.shipyard.e2e.yml
```

Comment:
Please keep them where they were, there's no guarantee consuming projects will have exact same file names

Reply:
I fixed this by parsing the `using` flag in both makefiles so that we can ensure we're always up to date. I am definitely not a MAKE expert so please let me know if this makes sense.

Reply:
Yes there's a bit of a "chicken and egg" situation here since the `_using` part is traditionally parsed in `Makefile.inc` but `SETTINGS` is expected to be set before including it, so we were actually silently not using `ovn` anywhere for a while.

Perhaps the best solution is to move `_using` parsing into `Makefile.dapper` as that one gets loaded way before, or just create some dedicated `Makefile.variables` file (to load any such shared variables).

Reply:
Done, I added the parsing of the `using` flag to `Makefile.dapper` I think it makes it much easier to understand